compose/fakes/client.go

```diff
@@ -0,0 +1,59 @@
+package fakes
+
+import composeapi "github.com/compose/gocomposeapi"
+
+type FakeComposeClient struct {
+	CreateDeploymentParams  composeapi.DeploymentParams
+	FoundDeployments        *[]composeapi.Deployment
+	DeprovisionDeploymentID string
+	SetScalingsParams       composeapi.ScalingsParams
+	GetRecipeID             string
+	GetRecipeErr            error
+	GetRecipeStatus         string
+}
+
+func NewFakeClient() (*FakeComposeClient, error) {
+	return &FakeComposeClient{}, nil
+}
+
+var (
+	deployment1 = &composeapi.Deployment{ID: "1", Name: "bd7a89b4-2a99-4ea2-a369-2ec42230af72"}
+	deployment2 = &composeapi.Deployment{
+		ID:         "2",
+		Name:       "3d4548fe-ea5e-4ad8-bd2d-a677c76b8275",
```

Comment:
This ID is also hardcoded in the tests. It might be clearer if the tests populated the expected deployments in `BeforeEach()`